src/loginServices.js

```diff
@@ -41,6 +42,11 @@ export function loadTranslations(store, locale) {
   // don't know how to talk to each other. sheesh.
   const region = locale.replace('-', '_');
 
+  // Update dir- and lang-attributes on the HTML element
+  // when the locale changes
+  document.documentElement.setAttribute('lang', parentLocale);
+  document.documentElement.setAttribute('dir', isRTL(parentLocale) ? 'rtl' : 'ltr');
```

Comment:
Hmmmm. I wonder if we should only update the `lang` and `dir` elements for supported locales, i.e. in the `then((response))...` block near the `store.dispatch(...)` calls rather than up above. In this implementation, somebody could submit a request with an unsupported locale and the display would flip to RTL but the labels would still remain English. That seems incorrect to me. 

What do you think?

Reply:
At first, I had the code within those blocks so that it only changed when that request resolved. 

I moved it since on page refresh it would load the English locale first and once the request resolved it would change the language and the direction. This caused the layout to flash which looked a bit buggy to me.

I was thinking that this approach would be fine since users probably wouldn't be able to change language to any non-existing languages (if they use the UI to change language).